twenty-first/src/storage/storage_vec/mod.rs

```diff
@@ -0,0 +1,523 @@
+//! Provides a DB backed Vector API that is thread-safe, cached, and atomic
+
+// We have split storage_vec into individual files, but for compatibility
+// we still keep everything in mod storage_vec.
+//
+// To accomplish that, we keep the sub modules private, and
+// add `pub use sub_module::*`.
+
+#![allow(missing_docs)]
+mod iterators;
+mod ordinary_vec;
+mod ordinary_vec_private;
+mod rusty_leveldb_vec;
+mod rusty_leveldb_vec_private;
+pub mod traits;
+
+pub type Index = u64;
+
+pub use {iterators::*, ordinary_vec::*, rusty_leveldb_vec::*};
+
+#[cfg(test)]
+mod tests {
+    use std::sync::Arc;
+
+    use crate::storage::level_db::DB;
+
+    use super::traits::*;
+    use super::*;
+
+    use itertools::Itertools;
+    use rand::{Rng, RngCore};
+    use std::collections::HashMap;
+
+    use leveldb::batch::WriteBatch;
+
+    // todo: delete fn
+    pub(super) fn get_test_db(destroy_db_on_drop: bool) -> Arc<DB> {
+        Arc::new(DB::open_new_test_database(destroy_db_on_drop, None, None, None).unwrap())
+    }
+
+    // todo: delete fn
+    fn open_test_db(path: &std::path::Path, destroy_db_on_drop: bool) -> Arc<DB> {
+        Arc::new(DB::open_test_database(path, destroy_db_on_drop, None, None, None).unwrap())
+    }
```

Comment:
Why do you want to delete these helper functions?

Reply:
I think that was because before they were doing something a bit more complicated, but now it's mostly done in DB::open_test_database().   They are probably still a fine shorthand though, so I may just drop the comment.

Reply:
comments removed.